yt/frontends/athena_pp/data_structures.py

```diff
@@ -153,13 +152,22 @@ def __init__(
         zrat = self._handle.attrs["RootGridX3"][2]
         self._nonuniform = xrat != 1.0 or yrat != 1.0 or zrat != 1.0
         self._magnetic_factor = get_magnetic_normalization(magnetic_normalization)
+
+        geom = self._handle.attrs["Coordinates"].decode("utf-8")
+        self.geometry = Geometry(geom_map[geom])
+        if geom_map[geom] == "cylindrical":
```

Comment:
This would be clearer as follow

```suggestion
        if self.geometry == "cylindrical":
```